nss-day-cohort-52/nashville-kennels-cocoa-foxes
# Animal discharge

Description:

Employee's can click on the discharge button and the animal will be deleted from the database. 

Related Issue Tickets:

Fixes #7

Steps to Test:

Provided that the currently logged-in user is an employee, navigate to the animals list using the nav bar. The user may then click on the drop down that is in each  animal box to view their individual card and a "discharge" button will appear at the bottom of the card. Clicking on the "discharge" button should result in a 200 DELETE Network call in the dev tools for the associated userId and the page will immediately refresh to the updated animal list.
